src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -43,6 +48,24 @@ public static Index parseIndex(String oneBasedIndex) throws IllegalValueExceptio
         }
         return Index.fromOneBased(Integer.parseInt(trimmedIndex));
     }
+    //@@ author arnollim
+    /**
+     * @param args into a String, which will be the filename which the .txt file will be saved as
+     * for e.g. if filename was the arg, then the file will be saved as filename.txt
+     * Leading and trailing whitespaces will be trimmed.
+     * @return "filename" for example.
+     * @throws IllegalValueException if there is no specified filepath.
+     */
+    public static String parseFilePath(String args) throws IllegalValueException {
+        final Matcher matcher = PRINT_ARGS_FORMAT.matcher(args.trim());
+        if (!matcher.matches()) {
+            throw new IllegalValueException(MESSAGE_INVALID_INDEX);
+        }
+        {
+            return matcher.group("filename");
+        }
+    }
```

Comment:
dont really understand the purpose of this function...? what are you parsing?

Reply:
I'm still trying to write a parser that refuses to take in the standard forbidden symbols in filenames, e.g. #./ and so on. So far I've not been successful yet, hence it seems that my parser does nothing yet